src/OnlineSales/Migrations/20230529131248_Commentable.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace OnlineSales.Migrations
+{
+    /// <inheritdoc />
+    public partial class Commentable : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "fk_comment_content_content_id",
+                table: "comment");
+
+            migrationBuilder.DropIndex(
+                name: "ix_comment_content_id",
+                table: "comment");
+
+            migrationBuilder.RenameColumn(
+                name: "content_id",
+                table: "comment",
+                newName: "commentable_id");
+
+            migrationBuilder.AddColumn<string>(
+                name: "commentable_type",
```

Comment:
@IgorKostyukov all existing records in the comments table should be updated with commentable_type = Content